grammar.js

```diff
@@ -54,6 +54,7 @@ module.exports = grammar({
     [$._primary_expression, $._array_destructing],
     [$._array_destructing, $.array_creation_expression],
 
+    [$.type_list],
```

Comment:
Is this conflict still needed?

Reply:
You're quite right, it's no longer needed. Good catch! Fix now